boards/mpfs-icicle-kit-es/bfn_defconfig

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
Change the file name to bni_delta_defconfig

Reply:
Changed to bni-delta_defconfig